_inc/client/admin.js

```diff
@@ -59,7 +59,24 @@ if ( 'undefined' !== typeof window && process.env.NODE_ENV === 'development' ) {
 
 render();
 
+async function renderUnifiedImporter( unifiedImporterElement ) {
+	const {
+		default: UnifiedImporter,
+	} = await import( /* @TODO actually split into another bundle..? */
```

Comment:
This is getting flagged by the [pre-commit check](https://github.com/Automattic/jetpack/blob/f024195cb6f208603c6da239431c5c5bc481f3a9/bin/pre-commit-hook.js#L95).  I tried to work around it, but haven't yet figured out how.